README.md

```diff
@@ -5,7 +5,7 @@ LazyStarter is a composable, component-based scaffolding framework.
 - **Language-Agnostic:** Templates can be in your programming language of your choice
 - **Do It Once, Get It Twice:** Write your components and we take care of creating the flags for you!
 - **Component-Based**
-- **
+-
```

Comment:
ehm, what happened here...? Whatever. Next time just use --allow-empty flag on your commit so that you can commit without changing files.